firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -44,7 +73,31 @@ public static FirebaseAppDistribution getInstance() {
    */
   @NonNull
   public Task<AppDistributionRelease> updateToLatestRelease() {
-    return Tasks.forResult(null);
+    Log.v("updateToLatestRelease", "called");
+    Log.v("updateToLatestRelease", currentActivity.getClass().getName());
+
+    TaskCompletionSource<AppDistributionRelease> taskCompletionSource =
+        new TaskCompletionSource<>();
+
```

Comment:
you can probably combine these two lines (dont need to assign SignInTester separately) - 
```
signInTester()
      .addOnSuccessListener(....
```